modules/fieldlinessequence/rendering/renderablefieldlinessequence.h

```diff
@@ -192,29 +209,25 @@ class RenderableFieldlinesSequence : public Renderable {
     // Index of the extra quantity to use for masking
     properties::OptionProperty _pMaskingQuantity;
```

Comment:
No need to have the p in the beginning to mark it as a property.  In the best case people shouldn't need to care whether it is a property or not